docs/pages/web/combobox.js

```diff
@@ -6,59 +6,26 @@ import docgen, { type DocGen } from '../../docs-components/docgen.js';
 import Page from '../../docs-components/Page.js';
 import GeneratedPropTable from '../../docs-components/GeneratedPropTable.js';
 import QualityChecklist from '../../docs-components/QualityChecklist.js';
-
+import SandpackExample from '../../docs-components/SandpackExample.js';
 import AccessibilitySection from '../../docs-components/AccessibilitySection.js';
+import main from '../../examples/combobox/main.js';
+import labels from '../../examples/combobox/labels.js';
+import localization from '../../examples/combobox/localization.js';
+import programmatic from '../../examples/combobox/programmatic.js';
+import uncontrolled from '../../examples/combobox/uncontrolled.js';
+import controlled from '../../examples/combobox/controlled.js';
+import tags from '../../examples/combobox/tags.js';
+import ref from '../../examples/combobox/ref.js';
+import subtext from '../../examples/combobox/subtext.js';
+import error from '../../examples/combobox/error.js';
 
 export default function ComboBoxPage({ generatedDocGen }: {| generatedDocGen: DocGen |}): Node {
   return (
-    <Page title="ComboBox">
-      <PageHeader
-        name="ComboBox"
-        description={generatedDocGen?.description}
-        defaultCode={`
-function ComboBoxExample(props) {
-  const PRONOUNS = [
-    'ey / em',
-    'he / him',
-    'ne / nem',
-    'she / her',
-    'they / them',
-    've / ver',
-    'xe / xem',
-    'xie / xem',
-    'zie / zem',
-  ];
-
-  const options = PRONOUNS.map((pronoun, index) => ({ label: pronoun, value: 'value'+index }));
+    <Page title={generatedDocGen?.displayName}>
+      <PageHeader name={generatedDocGen?.displayName} description={generatedDocGen?.description}>
+        <SandpackExample code={main} name="No image source" hideEditor previewHeight={320} />
```

Comment:
nit: I don't understand how this name relates to the code example?

Reply:
Didn't know they were a11y labels.

FIXED DONE